[Bug] The build binary package failed to run.

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.


### Environment

Mac

### EventMesh version

master

### What happened

build eventmesh binary package failed to run.

### How to reproduce

1.run ./gradlew clean jar dist && ./gradlew installPlugin
2.building binary packages
3.Unzip and run the binary package.

### Debug logs

```
2023-12-14 12:10:34,662 ERROR [main] EventMeshStartup(EventMeshStartup.java:55) - EventMesh start fail.
java.lang.NoClassDefFoundError: org/apache/eventmesh/retry/api/AbstractRetryer
	at java.lang.ClassLoader.defineClass1(Native Method) ~[?:1.8.0_311]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:756) ~[?:1.8.0_311]
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142) ~[?:1.8.0_311]
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:473) ~[?:1.8.0_311]
	at java.net.URLClassLoader.access$100(URLClassLoader.java:74) ~[?:1.8.0_311]
	at java.net.URLClassLoader$1.run(URLClassLoader.java:369) ~[?:1.8.0_311]
	at java.net.URLClassLoader$1.run(URLClassLoader.java:363) ~[?:1.8.0_311]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.8.0_311]
	at java.net.URLClassLoader.findClass(URLClassLoader.java:362) ~[?:1.8.0_311]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418) ~[?:1.8.0_311]
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:355) ~[?:1.8.0_311]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_311]
	at org.apache.eventmesh.runtime.boot.EventMeshHTTPServer.init(EventMeshHTTPServer.java:129) ~[eventmesh-runtime-1.9.0-release.jar:1.9.0-release]
	at org.apache.eventmesh.runtime.boot.EventMeshHttpBootstrap.init(EventMeshHttpBootstrap.java:48) ~[eventmesh-runtime-1.9.0-release.jar:1.9.0-release]
	at org.apache.eventmesh.runtime.boot.EventMeshServer.init(EventMeshServer.java:125) ~[eventmesh-runtime-1.9.0-release.jar:1.9.0-release]
	at org.apache.eventmesh.runtime.boot.EventMeshStartup.main(EventMeshStartup.java:40) [eventmesh-runtime-1.9.0-release.jar:1.9.0-release]
Caused by: java.lang.ClassNotFoundException: org.apache.eventmesh.retry.api.AbstractRetryer
	at java.net.URLClassLoader.findClass(URLClassLoader.java:387) ~[?:1.8.0_311]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418) ~[?:1.8.0_311]
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:355) ~[?:1.8.0_311]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_311]
	... 16 more
2023-12-14 12:10:34,669 pool-1-thread-1 DEBUG Stopping LoggerContext[name=4aa298b7, org.apache.logging.log4j.core.LoggerContext@6253c26]
2023-12-14 12:10:34,674 pool-1-thread-1 DEBUG Stopping LoggerContext[name=4aa298b7, org.apache.logging.log4j.core.LoggerContext@6253c26]...
2023-12-14 12:10:34,681 pool-1-thread-1 DEBUG Shutting down OutputStreamManager SYSTEM_OUT.false.false
2023-12-14 12:10:34,682 pool-1-thread-1 DEBUG OutputStream closed
2023-12-14 12:10:34,682 pool-1-thread-1 DEBUG Shut down OutputStreamManager SYSTEM_OUT.false.false, all resources released: true
2023-12-14 12:10:34,682 pool-1-thread-1 DEBUG Appender console stopped with status true
2023-12-14 12:10:34,682 pool-1-thread-1 DEBUG Log4j2 ConfigurationScheduler shutting down threads in java.util.concurrent.ScheduledThreadPoolExecutor@4328d5ab[Running, pool size = 1, active threads = 0, queued tasks = 1, completed tasks = 0]
2023-12-14 12:10:34,683 pool-1-thread-1 DEBUG Stopped XmlConfiguration[location=/Users/alone/eventmesh_test/test02/eventmesh-1.9.0-release/conf/log4j2.xml] OK
2023-12-14 12:10:34,683 pool-1-thread-1 DEBUG Stopped LoggerContext[name=4aa298b7, org.apache.logging.log4j.core.LoggerContext@6253c26] with status true
```

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [ ] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct) *

--- a/build.gradle
+++ b/build.gradle
@@ -317,6 +317,7 @@ subprojects {
                                    "eventmesh-metrics-api",
                                    "eventmesh-meta-api",
                                    "eventmesh-trace-api",
+                                   "eventmesh-retry-api",
                                    "eventmesh-runtime",
                                    "eventmesh-security-api",
                                    "eventmesh-protocol-api",

